### PR TITLE
Add filter configs in FOFB window

### PR DESCRIPTION
**Pull request by anacso17**

### CODE REVIEW

File: pyqt-apps/siriushla/si_ap_fofb/main.py [hunk 1]
```diff
@@ -362,6 +362,37 @@ def _setupDetailsWidget(self):
...
+                visrule = (
+                    '[{"name": "VisRule", "property": "Visible", ' +
+                    '"expression": "ch[0] == 2", "channels": ' +
+                    '[{"channel": "' +
+                    pref.substitute(propty='FOFBAccFilter-Sel') +
+                    '", "trigger": true}]}]')
+                sel_acc_filter = SiriusEnumComboBox(
+                    self, pref.substitute(propty = 'FOFBAccFilter-Sel'))
+                sts_acc_filter = SiriusLabel(
+                    self, pref.substitute(propty='FOFBAccFilter-Sts'))
```
Comment (ericonr) [inline pyqt-apps/siriushla/si_ap_fofb/main.py:377]: There was something weird with the "Custom" option where it extended the combo box beyond its original size. I don't know what caused it...

File: pyqt-apps/siriushla/si_ap_fofb/main.py [hunk 2]
```diff
@@ -362,6 +362,37 @@ def _setupDetailsWidget(self):
...
+                sts_acc_filter.rules = visrule.replace('==', '!=')
```
Comment (ericonr) [inline pyqt-apps/siriushla/si_ap_fofb/main.py:378]: Does `replace` make sense here if we are only using the version with the replacement?

**Review by xresende — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]

File: pyqt-apps/siriushla/si_ap_fofb/main.py (hunk 1 above)
Comment (anacso17) [reply pyqt-apps/siriushla/si_ap_fofb/main.py:370]: the previous commit fix this :)

File: pyqt-apps/siriushla/si_ap_fofb/main.py (hunk 2 above)
Comment (anacso17) [reply pyqt-apps/siriushla/si_ap_fofb/main.py]: removed :)

**Pull request merged**